dev/benchmarks/complex_layout/windows/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ set(BINARY_NAME "complex_layout")
 
 # Explicitly opt in to modern CMake behaviors to avoid warnings with recent
 # versions of CMake.
-cmake_policy(SET CMP0063 NEW)
+cmake_policy(VERSION 2.4...3.25)
```

Comment:
Have you tested with 3.25, or only with 3.24?

Reply:
I tested with 3.25